.env-sample

```diff
@@ -1 +1,4 @@
-PORT=
\ No newline at end of file
+PORT=
+DATABASE_URL=
+CLIENT_ID=
+CLIENT_SECRET=
```

Comment:
this may not be an issue now but i wonder, if we ever get to adding FB integration, i assume we'd have another set of client id and secret? what do you think about being explicit on the env var name like `GOOGLE_CLIENT_ID` etc.